### DISCUSSION TITLE
Audio Drop Outs

### DISCUSSION THREAD

**Discussion by Rogerwalker259** [AzuraCast/AzuraCast#4802] [General]
Hi Guys Problem I am getting is, random drop outs at the the end of random tracks. Drop outs always happen in the last 2 seconds of a  track. Track stops 1 or 2 seconds and then continues to the end. I am using the auto DJ on random play, problem can happen 4 or 5 times an hour, and sometimes does'nt happen for hours. When i run live stream runs perfectly. 
Anybody got any Ideas ?
Roger

**Comment by SpeedyCharly** [comment]
You seem to be running into buffer overruns... try increasing your buffer setting.
